docs/manual/index-checker.tex

```diff
@@ -156,12 +156,16 @@ \section{Lower bounds\label{index-lowerbound}}
 
 The Index Checker issues an error when
 a sequence is indexed by an integer that might be negative.
-It uses a type system (Figure~\ref{fig-index-int-types}) with the following
+The Lower Bound Checker uses a type system (Figure~\ref{fig-index-int-types}) with the following
 qualifiers:
 
 \begin{description}
 \item[\refqualclass{checker/index/qual}{Positive}]
   The value is 1 or greater, so it is not too low to be used as an index.
+  Note that this annotation is trusted by the Constant Value Checker,
```

Comment:
Right now it's not possible to run the Index Checker without running the Lower Bound Checker. We would need to have the Value Checker issue such a warning, which I think isn't feasible (do checkers have access to the list of checkers that will run after them?).